components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -137,6 +137,9 @@ public synchronized static APIGatewayManager getInstance() {
                 if (environment == null) {
                     continue;
                 }
+                if (debugEnabled) {
+                    log.debug(api.getId() + "publishing to the environment of " + environment);
```

Comment:
We can't print the environment object. Print the environment name

Reply:
https://github.com/wso2/carbon-apimgt/pull/9019/commits/29a2e4bba591e33476f9fcce64bd8668945123c4